core/src/main/scala/ml/dmlc/xgboost4j/scala/spark/XGBoostParams.scala

```diff
@@ -81,6 +82,25 @@ case object OpXGBoost {
     }
   }
 
+  implicit class RichBooster(val booster: Booster) extends AnyVal {
+    /**
+     * Converts feature score map into a vector
+     *
+     * @param featureVectorSize
+     * @return
+     */
+    def getFeatureScoreVector(featureVectorSize: Option[Int]): Vector = {
```

Comment:
1. why use `Vector` as return type for if you are doing to `toArray` anyways?
2. should we have a default value `featureVectorSize: Option[Int] = None`?

Reply:
@kinfaikan ^